[MIG] withhldings + perceptions to 18

jjscarafia opened the pull request.

jjscarafia committed:
[REF] l10n_ar_tax: merge _account_withholding and _withholding_ux
Commit inicial donde mezclamos ambos módulos en nuevo modulo l10n_ar_tax

jjscarafia committed:
[MIG] l10n_ar_tax: Migration to 18.0

jjscarafia committed:
[REF] main withholdings refactor

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-argentina/pull/939).

jjscarafia committed:
[REF] rename company_tax_ws fo l10n_ar_tax

jjscarafia committed:
[FIX] minor fixes

jjscarafia committed:
[IMP] l10n_ar_tax: implement payment pro pay withholdings

jjscarafia committed:
[FIX] depends

jjscarafia committed:
[IMP] add info of wth computation

jjscarafia committed:
[FIX] duplicate of payment

jjscarafia committed:
[IMP] move transfer report to specific module

Generalizamos el reporte de intenral transfer para que sea utilizable por otras localizaciones y mas facilmente heredable (por ej, agregado de data de cheques)

jjscarafia committed:
[IMP] re use native odoo receipt report

Si bien por ahora no reutiliza nada del reporte original nos parece mas prolijo heredarlo y eventualmente empezar a re-utilizar cosas

jjscarafia committed:
[ADD] sircar and api views

jjscarafia committed:
[FIX] l10n_ar_tax: to pay amount from checks
